UKHO.ExternalNotificationService.API/UKHO.ExternalNotificationService.API.FunctionalTests/Helper/EnsApiClient.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System.Net.Http;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using Newtonsoft.Json;
+using UKHO.ExternalNotificationService.API.FunctionalTests.Model;
+
+namespace UKHO.ExternalNotificationService.API.FunctionalTests.Helper
+{
+    public class EnsApiClient
+    {
+        static readonly HttpClient s_httpClient = new();
+        private readonly string _apiHost;
+
+        public EnsApiClient(string apiHost)
+        {
+            _apiHost = apiHost;
+        }
+
+        /// <summary>
+        /// Post Subscription request
+        /// </summary>
+        /// <param name="d365Payload"></param>
+        /// <param name="headerRequest">headerRequest, pass NULL to skip request header</param>
+        /// <returns></returns>
+        public async Task<HttpResponseMessage> PostEnsApiSubcriptionAsync([FromBody] D365Payload d365Payload, string headerRequest = null)
```

Comment:
Typo fix please:
```suggestion
        public async Task<HttpResponseMessage> PostEnsApiSubscriptionAsync([FromBody] D365Payload d365Payload, string headerRequest = null)
```

Reply:
Thanks for highlighting ,corrected spelling.